src/CMakeLists.txt

```diff
@@ -285,6 +299,33 @@ add_library(${LIB_STATIC_Z} STATIC IMPORTED GLOBAL)
 find_package(PythonLibs REQUIRED)
 include_directories(SYSTEM ${PYTHON_INCLUDE_DIRS})
 
+#
+# Python support
+#
+set (PY_SUPPORT_FILES
+     nupic/py_support/NumpyVector.cpp
+     nupic/py_support/PyArray.cpp
+     nupic/py_support/PyHelpers.cpp
+     nupic/py_support/PythonStream.cpp
+)
+
+message(STATUS "PYTHON_EXECUTABLE   = ${PYTHON_EXECUTABLE}")
+message(STATUS "PYTHON_INCLUDE_DIRS = ${PYTHON_INCLUDE_DIRS}")
+message(STATUS "PYTHON_LIBRARIES    = ${PYTHON_LIBRARIES}")
+
+#
+# Find and add capnp and kj and generate Cap'n Proto C++ files
+#
+set(CAPNPC_OUTPUT_DIR ${CMAKE_BINARY_DIR})
+set(CAPNPC_IMPORT_DIRS ${CAPNPC_IMPORT_DIRS} ${REPOSITORY_DIR}/src)
+find_package(CapnProto REQUIRED)
```

Comment:
Currently all of these things are supposed to be handled in `external/CMakeLists.txt` and `external/CapnProto.cmake`. Perhaps we can just update `external/CapnProto.cmake` to work with Windows? Also, make sure AppVeyor is passing cmake the directory to the root of the repo instead of `src` (`cmake ../..` instead of `cmake ../../src`).